src/components/Dropdown.tsx

```diff
@@ -27,12 +29,13 @@ export interface DropdownItemProps {
     onClick: (value?: any) => void;
 }
 
-interface DropdownProps {
+export interface DropdownProps {
     renderItem: (props: DropdownItemProps) => React.ReactNode;
     renderTrigger: (props: DropdownTriggerProps) => React.ReactNode;
     text?: string;
     value?: any;
     items?: any[];
+    searchFilter?: (item: any, query: string) => boolean;
```

Comment:
`searchFilter` not correct :( `searchQuery `or `query` would be better

Reply:
changed to `onSearchChange`